apps/dapp/src/constants/env/local.tsx

```diff
@@ -62,7 +61,8 @@ const env: Environment = {
     balancerV2:
       'https://api.thegraph.com/subgraphs/name/templedao/templedao-balancer-v2',
     // TODO: Will be deprecated
-    ramos: 'https://api.studio.thegraph.com/query/76011/temple-ramos/version/latest',
+    ramos:
+      'https://api.studio.thegraph.com/query/76011/temple-ramos/version/latest',
```

Comment:
We're not using the graph anymore - are these to be cleaned up?

Reply:
Good point, I will add removing unused env vars to the cleanup PR.